src/test/java/org/lsc/jmx/LscJmxTest.java

```diff
@@ -74,6 +75,7 @@
  * This test covers all the JMX capabilities
  * @author Sebastien Bahloul &lt;seb@lsc-project.org&gt;
  */
+@Disabled
```

Comment:
Why is it disabled? Is JMX not supported any more?

If JMX is still supported, I think we should have another test instead of just disabling it.